CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns MX on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Distributes functions with `CREATE FUNCTION` command
+
+* `citus_shard_indexes_on_worker` shows all local shard indexes regardless of
```

Comment:
The two lines are less interesting, so could go somewhere bottom in the newly added feature section.